ui/src/app/shared/requirements/list/requirements.list.html

```diff
@@ -45,7 +45,15 @@
                             <input name="value-os-architecture" auto-complete autocomplete="off" type="text" [source]="suggestWithOSArch" [min-chars]="0" [(ngModel)]="r.value" (ngModelChange)="change()" [no-match-found-text]="false">
                         </div>
                         <div *ngIf="!loading && r.type === 'model' && workerModels">
-                            <input name="value-workermodel" auto-complete autocomplete="off" type="text" [source]="suggestWithWorkerModel" [min-chars]="0" [(ngModel)]="r.value" (ngModelChange)="change()" [no-match-found-text]="false">
+                            <div class="ui grid">
+                                <div class="ui fourteen wide column">
+                                        <input name="value-workermodel" auto-complete autocomplete="off" type="text" [source]="suggestWithWorkerModel" [min-chars]="0" [(ngModel)]="r.value" (ngModelChange)="change()" [no-match-found-text]="false">
+                                </div>
+                                <div class="ui two wide column valign">
+                                    <span *ngIf="getWorkerModel(r.value)?.is_official" [smDirTooltip]="'worker_model_official_tooltip' | translate" smDirPosition="top center"><i class="check circle outline icon large green"></i></span>
+                                    <span *ngIf="getWorkerModel(r.value)?.is_deprecated" [smDirTooltip]="'worker_model_deprecated_tooltip' | translate" smDirPosition="top center"><i class="exclamation circle icon large orange"></i></span>
```

Comment:
if too many error with a worker model (5) -> status is disabled. It would be useful to user to see that the selected worker model is disabled in job requirements. So, it should be great to have an icon for the status disabled or not, as deprecated and official.